source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,166 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections using **read operations**.
+
+Read operations allow you to do the following:
+
+- Retrieve documents from your collections by using :ref:`find operations <retrieve-find>`
+- Perform transformations on documents in your collections by using :ref:`aggregation operations <retrieve-aggregation>`
+
+Sample Data
+~~~~~~~~~~~
+
+Run the following snippet to load the documents into the ``ratings``
+collection of the ``tea`` database:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/retrieve.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+Each document contains a rating for a type of tea, which corresponds to
+the ``name`` and ``rating`` fields.
+
+.. note::
+
+   Each example truncates the ``ObjectID`` value since the driver
+   generates them uniquely.
+
+.. _retrieve-find:
+
+Find Operations
+---------------
+
+Use **find operations** to retrieve data from MongoDB. Find operations
+consist of the ``Find()`` and ``FindOne()`` functions.
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+The ``Find()`` function expects you to pass a ``Context`` type and a
+query filter. The function returns *all* documents that match the filter as a
+``Cursor`` type. If you pass an empty filter, the function returns *all*
+documents in the collection.
+
+.. To learn how to access data in a cursor, see the :doc:`Cursor
+.. </fundamentals/crud/read-operations/cursor>` guide.
+
+Find One Document
+~~~~~~~~~~~~~~~~~
+
+The ``FindOne()`` function expects you to pass a ``Context`` type and a
+query filter. The function returns *the first document* that matches the
+filter as a ``SingleResult`` type. If you pass an empty filter, the
+function returns *a random document* in the collection.
```

Comment:
I interpret "a random document" to mean a randomly selected document from the collection. I believe the server behavior is to return the first document it scans. 

https://docs.mongodb.com/manual/reference/glossary/#std-term-natural-order describes the natural ordering of stored documents.

I suggest removing the last sentence. An empty filter matches all documents. An empty filter does not need to be called out as a special case.